docs/gravitino-server-config.md

```diff
@@ -42,11 +42,11 @@ You can also specify filter parameters by setting configuration entries of the f
 
 ### Storage configuration
 
+Currently, Gravitino only supports JDBC backend, and the default implementation is H2, if you are going to use H2 in the production environment, Gravitino will not guarantee the data consistency and durability. We recommend using MySQL as the backend database.  
```

Comment:
It's easy to test the Gravitino locally.

Reply:
Do you mean we can add the sentence `It's easy to test the Gravitino locally.` here?

Reply:
Yes. Just give an example. We need explain why default implement is H2. 
Correct my fault. We should use `the Gravitino server` or `Gravitino`.